server/scripts/import-tests/index.js

```diff
@@ -241,8 +241,8 @@ const ariaAtImport = {
 
         // checking to see if unique testPlan row
         const testPlanResult = await client.query(
-            'SELECT id, "sourceGitCommitHash" FROM "TestPlan" WHERE "sourceGitCommitHash"=$1 and "exampleUrl"=$2',
-            [commitHash, exampleUrl]
+            'SELECT id, "sourceGitCommitHash" FROM "TestPlan" WHERE "sourceGitCommitHash"=$1 and parsed ->> \'directory\'=$2',
+            [commitHash, exampleDir]
```

Comment:
I'm assuming this is the bug fix you mentioned? This definitely deserves a comment!